OrbitCore/SamplingProfiler.cpp

```diff
@@ -284,6 +297,26 @@ void SamplingProfiler::ResolveCallstacks() {
   }
 }
 
+//-----------------------------------------------------------------------------
+unsigned int SamplingProfiler::GetCountOfFunction(uint64_t function_address) {
+  auto addresses_of_functions_itr =
+      m_FunctionAddressToExactAddresses.find(function_address);
+  if (addresses_of_functions_itr == m_FunctionAddressToExactAddresses.end()) {
+    return 0;
+  }
+  unsigned int result = 0;
+  CHECK(GetGenerateSummary());
+  const ThreadSampleData& summary = GetSummary();
+  auto function_addresses = (*addresses_of_functions_itr).second;
```

Comment:
This is making of copy of the stored std::set. Is this intended? Or might be a const-ref enough?

Reply:
Made it a const ref.